llvm/lib/Target/X86/X86WinEHState.cpp

```diff
@@ -363,6 +363,16 @@ void WinEHStatePass::emitExceptionRegistrationRecord(Function *F) {
     Instruction *T = BB.getTerminator();
     if (!isa<ReturnInst>(T))
       continue;
+
+    // Back up to any preceding musttail call, the de-facto terminator.
+    Instruction *Prev = T->getPrevNonDebugInstruction();
```

Comment:
Let's use `BB.getTerminatingMustTailCall()` to share code and avoid edge cases.

Reply:
Thanks! I didn't know about that one.